src/forms/forms.controller.ts

```diff
@@ -28,6 +31,7 @@ export class FormsController {
         type: FormResponse,
         isArray: true,
     })
+    @Roles(AppRoles.Admin)
```

Comment:
does role guard need to to be added under Roles decorator?
```suggestion
    @Roles(AppRoles.Admin)
    @UseGuard(RoleGuard)
```